utils/chaos_ai/README.md

```diff
@@ -0,0 +1,30 @@
+# aichaos
+Enhancing Chaos Engineering with AI-assisted fault injection for better resiliency and non-functional testing.
+
+
+## Installing the dependencies
```

Comment:
@sandeephans Also need to add instructions on running containerized version with both docker and podman support which you added. Thoughts?